draft-ietf-rats-architecture.md

```diff
@@ -1226,57 +1227,64 @@ The following example illustrates a hypothetical Passport Model
+        |---Evidence--------------------->|               |
+        |   {Nonce1, time(EGa)-time(VCa)} |               |
+        |                              time(RGv)          |
+        |<--Attestation Result------------|               |
+        |   {time(RXv)-time(RGv)}         |               |
         ~                                                 ~
         |                                                 |
-        |<---Nonce2------------------------------------time(NS')
-     time(RR)
-        |----Attestation Result{time(RX)-time(RG)}---->time(RA)
-        |    Nonce2, time(RR)-time(EG)                    |
+        |<--Nonce2-------------------------------------time(NSr)
+     time(RRa)
+        |---Attestation Result{time(RXv)-time(RGv)}--->time(RAr)
+        |   Nonce2, time(RRa)-time(EGa)                   |
         ~                                                 ~
         |                                                 |
-        |                                              time(OP)
+        |                                              time(OPr)
 ~~~~
 
 In this example solution, the Verifier can check whether the Evidence is
-fresh at time(RG) by verifying that `time(RG) - time(NS) < Threshold`.
+fresh at time(RGv) by verifying that `time(RGv)-time(NSv) < Threshold`.
 
 The Verifier cannot, however, simply rely on a Nonce to
 determine whether the value of a claim is recent, since the claim value
 might have been generated long before the nonce was sent by the Verifier.
 However, if the Verifier decides that the Attester can be trusted to
-correctly provide the delta time(EG)-time(VG), then it can determine recency
-by checking `time(RG)-time(NS) + time(EG)-time(VG) < Threshold`.
+correctly provide the delta time(EGa)-time(VCa), then it can determine recency
+by checking `time(RGv)-time(NSv) + time(EGa)-time(VCa) < Threshold`.
 
 Similarly if, based on an Attestation Result from a Verifier it trusts,
 the Relying Party decides that the Attester can be trusted to correctly
 provide time deltas, then it can determine whether the Attestation
 Result is fresh by checking
-`time(OP) - time(NS') + time(RR)-time(EG) < Threshold`.
-Although the Nonce2 and time(RR)-time(EG) values cannot be inside
+`time(OPr)-time(NSr) + time(RRa)-time(EGa) < Threshold`.
+Although the Nonce2 and time(RRa)-time(EGa) values cannot be inside
 the Attestation Result, they might be signed by the Attester such
 that the Attestation Result vouches for the Attester's signing
 capability.
 
 The Relying Party must still be careful, however, to not allow continued
 use beyond the period for which it deems the Attestation Result to remain
 valid.  Thus, if the Attestation Result sends a validity lifetime
-in terms of time(RX)-time(RG), then the Relying Party can check
-`time(OP) - time(NS') < time(RX)-time(RG)`.
+in terms of time(RXv)-time(RGv), then the Relying Party can check
+`time(OPr)-time(NSr) < time(RXv)-time(RGv)`.
```

Comment:
Here is a little weird to me. Because even when the actual time(NSr) > actual time(RXv), 'time(OPr)-time(NSr) < time(RXv)-time(RGv)' may still be true. If so this comparsion makes no sense.
One better expression here, I think, is that 'time(OPr)-time(NSr) + time(RRa)-time(EGa) < time(RXv)-time(RGv)'.